src/tls/keypair.h

```diff
@@ -410,6 +410,12 @@ namespace tls
       return {first, buf + buf_size};
     }
 
+    // TODO: This should be removed
+    mbedtls_pk_context* get_raw_context() const
```

Comment:
Why?

Reply:
This was here before. I just moved it from the child class `KeyPair` to the parent class `PublicKey`.